test/math.spec.ts

```diff
@@ -0,0 +1,490 @@
+import { assert } from 'chai';
```

Comment:
One of the things we're trying to do in the larger refactor is remove duplication in tests. Where tests are very similar can you deduplicate them to make it where you're just passing in different inputs to the same code?